packages/react/src/Caret.tsx

```diff
@@ -109,9 +90,17 @@ function Caret(props: CaretProps) {
       role="presentation"
     >
       <g transform={transform}>
-        <path d={triangle} fill={theme?.colors.canvas.default} />
-        <path d={triangle} fill={bg} />
-        <path d={line} fill="none" stroke={borderColor} strokeWidth={borderWidth} />
+        <path d={triangle} className={classes.Background} />
+        <path d={triangle} style={{fill: props.bg}} className={classes.Background} />
+        <path
+          d={line}
+          fill="none"
+          style={{
+            stroke: props.borderColor,
+            strokeWidth: props.borderWidth,
```

Comment:
Would these values still work if a styled system value is provided? (e.g. `border.default`)

And I guess do we care? 😅

Reply:
ah yeah, looking at the snapshot updates looks like something isn't right